Scripts/manage-version.sh

```diff
@@ -18,19 +18,9 @@ OUTPUT_GREEN="\033[32m"
 OUTPUT_BOLD="\033[1m"
 
 # PList files
```

Comment:
A bit nitpicky (sorry!), but this would be the best moment to update the comment and the name of variables `buildPLists` and `intPLists` below.